book/src/api-reference/jsonrpc-api.md

```diff
@@ -149,6 +150,34 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "
 {"jsonrpc":"2.0","result":0,"id":1}
 ```
 
+### getBlockConfidence
+
+Returns confidence for particular block
+
+#### Parameters:
+
+* `u64` - fork, identified by slot id
+
+#### Results:
+
+The result field will be an array with two fields:
+
+* Confidence
+  * `null` - Unknown fork
+  * `object` - BankConfidence
+    * `array` - confidence, array of u64 integers logging the amount of stake in the cluster that has voted on the block at a particular depth from 0 to `MAX_LOCKOUT_HISTORY`
+* 'integer' - total stake in the cluster
```

Comment:
total stake *in lamports, at that block*?

Reply:
I don't think that's quite right. It's the total stake the validator currently sees in the cluster, not the stake that was delegated at the time of the block. Can you thin of a better way to put that?

Reply:
how about: `total active stake, in lamports, of the current epoch`

Reply:
Definitely better!